.github/workflows/test-suite-desktop-e2e-pr.yml

```
@@ -81,55 +88,41 @@ jobs:
-        if: steps.check-previous-runs.outputs.skip_tests != 'true'
         uses: ./.github/actions/run-e2e-tests
         with:
           project: "desktop"
-          containers: ${{ matrix.CONTAINERS }}
+          containers: "trezor-user-env-unix bitcoin-regtest"
           workspace-dependencies: "@trezor/suite-desktop"
           test-group: ${{ matrix.TEST_GROUP }}
+          fail-fast: 3
           currents-project-id: "4ytF0E"
           currents-record-key: ${{ secrets.CURRENTS_RECORD_KEY }}
+          currents-api-key: ${{ secrets.CURRENTS_API_KEY }}
```

actionlint (1.7.4)

input "currents-api-key" is not defined in action "Run Playwright E2E Tests" defined at "./.github/actions/run-e2e-tests". available inputs are "additional-grep", "base-url", "containers", "currents-project-id", "currents-record-key", "currents-tags", "e2e-passphrase", "fail-fast", "project", "test-group", "workspace-dependencies"

(action)